cmd/kubectl.lima

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+set -eu
+: "${LIMA_INSTANCE:=}"
+: "${KUBECTL:=kubectl}"
+
+if [ -z "$LIMA_INSTANCE" ]; then
+  if [ "$(limactl ls -f '{{.Status}}' k3s 2>/dev/null)" = "Running" ]; then
+    LIMA_INSTANCE=k3s
+  elif [ "$(limactl ls -f '{{.Status}}' k8s 2>/dev/null)" = "Running" ]; then
+    LIMA_INSTANCE=k8s
+  else
+    echo "no running instance found, run \`limactl start\` to start an instance" >&2
```

Comment:
Maybe include a hint that we are only looking for instance names `k3s` and `k8s`, and tell the user to set `LIMA_INSTANCE` if they use any other?